src/commtrack.js

```diff
@@ -282,7 +297,7 @@ define([
                     lstring: 'Destination Case',
                     visibility: 'visible',
                     presence: 'required',
-                    widget: setValueWidget,
+                    widget: widgets.xPath,
                     xpathType: "generic",
                     help: 'XPath expression for the case ID receiving the transaction. Leave blank if unknown or not applicable.',
```

Comment:
Not related to this PR, but I just realized the "Leave blank if unknown..." part here should be removed since these are now required for the Transfer question type. Same above.